### PR TITLE
Streaming plugin

### PR DESCRIPTION
**Pull request by vogdb**

### CODE REVIEW

**Comment by vogdb** [comment]
@zazabe please review

**Comment by zazabe** [comment]
- doc: `{int}` seems not a valid type, `{number}` should be used instead
- I would rather put the `createAnswer` (`startMediaStreaming`) inside the `watch` request, because `connect` could execute a `switch`, which has no `jsep` in response i suppose.   
 Maybe `startMediaStreaming` could even be "private".

**Comment by vogdb** [comment]
> - doc: `{int}` seems not a valid type, `{number}` should be used instead

I wanted to emphasise the fact that this id should be natural number.

> - I would rather put the `createAnswer` (`startMediaStreaming`) inside the `watch` request, because `connect` could execute a `switch`, which has no `jsep` in response i suppose.   

ok

> - Maybe `startMediaStreaming` could even be "private".

Why? It is a shortcut to the main logic of this plugin.

**Comment by vogdb** [comment]
@zazabe please look

File: README.md [hunk 1]
```diff
@@ -330,6 +331,81 @@ The library is available for Node and Browser environment. In Browser it is decl
...
+    * `mountpointId` int
+    * `options` Object. see JSDocu.
+
+ * `plugin.recording(mountpointId, [options])`
+
+    Requests to start or stop recording on the mountpoint.
+    * `mountpointId` int
+    * `options` Object. see JSDocu.
+
+ * `plugin.startMediaStreaming(jsep, [answerOptions])`
```
Comment (zazabe) [inline README.md]: remove it?

File: src/webrtc/plugin/streaming-plugin.js [hunk 2]
```diff
@@ -0,0 +1,225 @@
+var Promise = require('bluebird');
+var Helpers = require('../../helpers');
+var Plugin = require('../../plugin');
+var MediaEntityPlugin = require('../media-entity-plugin');
+
+function StreamingPlugin() {
+  StreamingPlugin.super_.apply(this, arguments);
+
+  /** @type {int} */
```
Comment (zazabe) [inline src/webrtc/plugin/streaming-plugin.js]: `{number}` ?

**Pull request merged**